SwiftOverpass/SwiftOverpass.swift

```diff
@@ -148,15 +118,16 @@ public final class OverpassApi {
      - parameter completion: A completion handler.
      */
     public func fetch(_ queries: [OverpassQuery], verbosity: OutputVerbosity? = nil, order: OutputOrder? = nil, completion: @escaping CompletionClosure) {
-        let requestQuery = makeXmlDocumentForRequestBody(queries: queries, verbosity: verbosity, order: order)
-        let parameters: [String : String] = ["data" : requestQuery.xmlCompact]
+        let builder = XMLQueryBuilder(queries: queries, verbosity: verbosity, order: order, timeout: timeout, elementLimit: elementLimit)
```

Comment:
By moving this private functionality to a dedicated object, we will be able to improve the testability. Thanks!